dataset/dictionary/domains.json5

```
@@ -256,62 +256,74 @@
   {
     en: "Violet Court",
     ja: "菫色ノ庭",
+    zhCN: "堇色之庭",
     pronunciationJa: "すみれいろノにわ",
     tags: [ "inazuma", "domain" ],
   },
   {
     en: "Court of Flowing Sand",
     ja: "砂流ノ庭",
+    zhCN: "砂流之庭",
     pronunciationJa: "さりゅうノにわ",
     tags: [ "inazuma", "domain" ],
   },
   {
     en: "Empty Boat of a Thousand Gates",
     ja: "千門の虚舟",
+    zhCN: "千门虚舟",
     tags: [ "inazuma", "domain" ],
   },
   {
     en: "Shakkei Pavilion",
     ja: "借景ノ館",
+    zhCN: "借景之馆",
     pronunciationJa: "しゃっけいノやかた",
     tags: [ "inazuma", "domain" ],
   },
   {
```

Let's add Momiji-Dyed Court here. Can you add the Chinese translation?

```suggestion
  {
    en: "Momiji-Dyed Court",
    ja: "紅葉ノ庭",
    zhCN: "",
    pronunciationJa: "もみじノにわ",
    tags: [ "inazuma", "domain" ],
  },
  {
```